assets/css/styles.css

```diff
@@ -30,6 +26,19 @@ h1 {
   color: #fff;
 }
 
+.btn:hover {
```

Comment:
- Provide the requested design. Add to your hover effect a color for your font inside your button.

Reply:
I did. there is a span inside the button.

Reply:
Kindly add the 

required design to your button submit in your styleguide(Figma).

When you hover the button the design is missing the text.

Yours:

![Screen Shot 2021-11-30 at 17 57 19](https://user-images.githubusercontent.com/22258274/144147328-c654d640-3685-45df-9f13-1a3f1b5e0553.png)

Styleguide:
![Screen Shot 2021-11-30 at 18 02 45](https://user-images.githubusercontent.com/22258274/144147480-fd2ae78e-0fa4-4491-a0e6-47760143ef9c.png)